assets/src/stories-editor/components/right-click-menu/index.js

```diff
@@ -60,7 +61,7 @@ const RightClickMenu = ( props ) => {
 	const blockClientIds = castArray( clientIds );
 	const firstBlockClientId = blockClientIds[ 0 ];
 	const block = getBlock( firstBlockClientId );
-	const isPageBlock = block ? 'amp/amp-story-page' === block.name : false;
+	const isPageBlock = isStoryBlock( firstBlockClientId );
```

Comment:
The function and the variable are inconsistently named here. Which is it?

Reply:
No they are not.

Reply:
Fixed https://github.com/ampproject/amp-wp/pull/3246/commits/ce6b984e34ce14bbd74859fd9c037066cab8799f